src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -267,10 +298,13 @@ export class MgtPersonCard extends MgtTemplatedComponent {
 
     person = person || this.personDetails;
     const userPerson = person as MicrosoftGraph.User;
+    const personPerson = person as MicrosoftGraph.Person;
+    const contactPerson = person as MicrosoftGraph.Contact;
 
     // Chat
     let chat: TemplateResult;
-    if (userPerson.userPrincipalName) {
+
+    if (userPerson.userPrincipalName || personPerson.userPrincipalName) {
```

Comment:
This is not necessary since it's the same property name (`userPrincipalName`). The compiled js would essentially end up being: `if (person.userPrincipalName || person.userPrincipalName)`

```suggestion
    if (userPerson.userPrincipalName) {
```